docker-compose.yml

```diff
@@ -70,7 +70,7 @@ services:
         hard: 10032
   postgres:
     <<: *restart_policy
-    image: "postgres:9.6"
+    image: "postgres:9.6-alpine"
```

Comment:
Let's do `postrges` on a separate PR (as we may need to punt on it and we already need to change the `wal2json` setup.

Reply:
```suggestion
    image: "postgres:9.6"
```

Reply:
Done, thank you for your patience !